test/e2e/create_test.go

```diff
@@ -31,31 +31,51 @@ var _ = Describe("Podman create", func() {
 	})
 
 	It("podman create container based on a remote image", func() {
-		session := podmanTest.Podman([]string{"create", "-q", BB_GLIBC, "ls"})
+		session := podmanTest.Podman([]string{"create", BB_GLIBC, "ls"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(ExitCleanly())
+		Expect(session).Should(Exit(0))
+		Expect(session.ErrorToString()).To(ContainSubstring("Trying to pull " + BB_GLIBC))
+		Expect(session.ErrorToString()).To(ContainSubstring("Writing manifest to image destination"))
+
 		Expect(podmanTest.NumberOfContainers()).To(Equal(1))
 	})
 
 	It("podman container create container based on a remote image", func() {
-		containerCreate := podmanTest.Podman([]string{"container", "create", "-q", BB_GLIBC, "ls"})
+		containerCreate := podmanTest.Podman([]string{"container", "create", BB_GLIBC, "ls"})
```

Comment:
Can we remove this test? It seems a waste to test the same for both `podman create` and `podman container create` especially when pulling an image as this causes unnecessary flakes and is slow.

If we truly want to test `podman container create` (which is already used in many other tests) then we should test it without having to pull a new image.

Reply:
With pleasure!